ServerCore/Pages/Submissions/Index.cshtml.cs

```diff
@@ -126,7 +126,7 @@ public async Task<IActionResult> OnPostAsync(int puzzleId, string submissionText
 
                 AnswerToken = submission.SubmissionText;
             }
-            else if (submission.Response == null)
+            else if (submission.Response == null && Event.IsAnswerSubmissionActive)
             {
                 // We also determine if the puzzle should be set to email-only mode.
                 if (IsPuzzleSubmissionLimitReached(
```

Comment:
Does this case also cover lockouts? It looks like it might, which is good

Reply:
Looks like you're right.